packages/jest-types/__typetests__/expect.test.ts

```diff
@@ -217,6 +217,38 @@ expectType<void>(expect(jest.fn()).toBeCalledWith('value', 123));
 expectType<void>(expect(jest.fn()).toHaveBeenCalledWith());
 expectType<void>(expect(jest.fn()).toHaveBeenCalledWith(123));
 expectType<void>(expect(jest.fn()).toHaveBeenCalledWith(123, 'value'));
+expectType<void>(expect(jest.fn()).toHaveBeenCalledWith(123, 'value'));
+
+/**
+ * type inference for "CalledWith" matchers parameters
+ */
+expectError(expect(jest.fn<(a: string) => void>()).toHaveBeenCalledWith(123));
+expectError(
+  expect(jest.fn<(a: string) => void>()).toHaveBeenNthCalledWith(123),
```

Comment:
First arg here is number of a call

```suggestion
  expect(jest.fn<(a: string) => void>()).toHaveBeenNthCalledWith(1, 123),
```

Reply:
Or was this intentional? Because it makes to have a test like that one too

Reply:
thanks, fixed